pandora-client-web/src/components/common/select/select.tsx

```diff
@@ -6,9 +6,15 @@ export interface SelectProps extends Omit<DetailedHTMLProps<SelectHTMLAttributes
 		currentTarget: HTMLSelectElement;
 		target: HTMLSelectElement;
 	}): void;
+	noScrollChange?: boolean;
 }
 
-export const Select = React.forwardRef<HTMLSelectElement, SelectProps>(function Select({ children, onChange, ...props }, ref): ReactElement {
+export const Select = React.forwardRef<HTMLSelectElement, SelectProps>(function Select({
```

Comment:
I was considering this the last time I made the PR but eventually, I decided against it, this property basically makes this whole element do nothing and work as a simple `<select>` element

we do not use this property dynamically so I see no benefit from adding this over simply just using `<select>`

Reply:
I agree, but I'm looking forward to the future: It is not unlikely we will add more features to the `Select` element, like being able to work with non-trivial values. For that case we want to reuse it in as many places as possible to be consistent. Also it gives a precedent for people that copy existing code to make new one.